DirectProgramming/DPC++FPGA/Tutorials/Features/prototype/CSR_and_streaming_interfaces/src/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# To see a Makefile equivalent of this build system:
+# https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC++/ProjectTemplates/makefile-fpga
+
+set(SOURCE_FILE CSR_and_streaming_interfaces.cpp)
+set(TARGET_NAME CSR_and_streaming_interfaces)
+set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+set(SIMULATOR_TARGET ${TARGET_NAME}.fpga_sim)
+
+# FPGA device selection
+if(NOT DEFINED FPGA_BOARD)
+    set(FPGA_BOARD "Arria10")
+    message(STATUS "FPGA_BOARD was not specified.\
+                    \nConfiguring the design to run on the default FPGA device ${FPGA_BOARD} (Intel(R) Arria(R) 10 FPGAs). \
+                    \nPlease refer to the README for information on board selection.")
+else()
+    message(STATUS "Configuring the design to run on FPGA device ${FPGA_BOARD}")
+endif()
+
+# This is a Windows-specific flag that enables exception handling in host code
+if(WIN32)
```

Comment:
This is not needed, please remove the WIN_FLAG entirely

Reply:
I see that other samples do still have this flag - is it safe to remove?